miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/api/AppInfoApi.kt

```diff
@@ -8,16 +8,17 @@ import retrofit2.http.Query
 
 internal interface AppInfoApi {
 
-    @GET("host/{hostappId}/miniapps")
+    @GET("host/{hostappId}/miniapps/{testPath}")
```

Comment:
So this would add an extra `/` on the end of the path for the normal published endpoints? Maybe you could encapsulate the `/` in the value passed to `testPath`? i.e. passing `/test`.

Reply:
I tested with `/test`, because it's going be retrofit api path, it will be encoded as `%2Ftest`. I'm not sure if it's okay for an api call so I will keep it like this for safe 😄  (an extra `/` in the end of the normal published endpoint).

Reply:
Ah, okay that makes sense.